frontend/benefit/applicant/src/components/applications/forms/application/consentViewer/ConsentViewer.tsx

```diff
@@ -0,0 +1,54 @@
+import { TermsProp, TextProp } from 'benefit/applicant/types/application';
+import { DynamicFormStepComponentProps } from 'benefit/applicant/types/common';
+import { Link } from 'hds-react';
+import * as React from 'react';
+import { $Checkbox } from 'shared/components/forms/fields/Fields.sc';
+import { $GridCell } from 'shared/components/forms/section/FormSection.sc';
+import { useTheme } from 'styled-components';
+
+import { useConsentViewer } from './useConsentViewer';
+
+const ConsentViewer: React.FC<DynamicFormStepComponentProps> = ({ data }) => {
+  const { textLocale, cbPrefix, t } = useConsentViewer();
+  const theme = useTheme();
```

Comment:
Maybe we can check the `data` here, or better yet we can check it outside the component.
```suggestion
  const theme = useTheme();
  
  if (!data) return null;
```